tests/translate/__init__.py

```diff
@@ -21,9 +26,6 @@
 from .translate_delnflux import TranslateDelnFlux, TranslateDelnFlux_2
 from .translate_divergencedamping import TranslateDivergenceDamping
 from .translate_dyncore import TranslateDynCore
-from .translate_fill4corners import TranslateFill4Corners
```

Comment:
Please don't rename the test class functions -- these are meant to correspond directly to the serialized savepoint names, it is super confusing if this pattern is broken. Actually, does the replacement, FillCornersCells actually run? I don't see how it could.